### DISCUSSION TITLE
Google chat / github integration for InboxSDK

### DISCUSSION THREAD

**Discussion by fssfnc** [InboxSDK/InboxSDK#1236] [Q&A]
Hi everyone,
I'm trying to configure google chat integration (https://workspace.google.com/marketplace/app/github_for_google_chat/536184076190) to receive updates in chat about new InboxSDK releases. In a configuration step I'm asked to insert github repo details in format "owner/repo", can you suggest what to write? 
Thank you

**Comment by aleemstreak** [comment]
InboxSDK/InboxSDK

**Comment by fssfnc** [comment]
Hi,
Trying to add the repository using InboxSDK/InboxSDK I get the error below and the indications given in the link are not helpful in solving the problem. Could it be that there are Oauth access restrictions for this repository?

Thank you

_Failed to save notification settings for InboxSDK/InboxSDK. If this is a personal repository, only the owner can create subscriptions. If this is an organization repository, only the organization owner can create subscriptions, and the Google Chat GitHub app must be [authorized to access the organization](https://www.google.com/url?q=https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/requesting-organization-approval-for-oauth-apps&source=chat&ust=1734507500113000&usg=AOvVaw0BxTOIAaU2PslzxwravqZk)._

**Comment by aleemstreak** [comment]
I think the error message is pretty explanatory.
ᐧ

**Reply by fssfnc** [reply]
Perhaps I am too dumb to fully understand, so forgive my insistence.
Trying to follow the pretty explanatory message:
If this repository is registered as personal (I guess not) then no way.
Assuming that this is an organisation repository named InboxSDK then I follow the steps indicated and get to point 4
(_In the list of applications, click the name of the OAuth app you'd like to request access for_) but the next step 5 (_Next to the organisation you'd like the OAuth app to access, click Request access_) is inapplicable because no organisation appears to me on that page to request access for, just a button to revoke access.

Thank you
